stats/worker_commands.go

```diff
@@ -144,22 +144,19 @@ func (cmd *unregisterViewReq) handleCommand(w *worker) {
 // subscribeToViewReq is the command to subscribe to a view.
 type subscribeToViewReq struct {
 	v   *View
-	c   chan *ViewData
 	err chan error
```

Comment:
Should this still be a chan error or just a normal error?

Reply:
I explained you the worker model and in order to communicate with that worker, we use channels. The error channel is the communication channel for the return value.